lib/iris/tests/unit/fileformats/netcdf/test_Saver.py

```diff
@@ -158,12 +158,13 @@ def test_big_endian(self):
     def test_zlib(self):
         cube = self._simple_cube('>f4')
         with mock.patch('iris.fileformats.netcdf.netCDF4') as api:
+            api.default_fillvals = {'f4': 12345.}
```

Comment:
I've had to modify this test because the `createVariable` call now passes the default fill value for the data type. Since the `netCDF4` module is mocked out, it turns out that the value passed for the `fill_value` argument is a numpy array. I'm guessing that this doesn't play well with `mock.ANY` (it probably tries to do an equality test somewhere along the line, which causes numpy to raise an exception), because the test fails unless I make sure there is an actual value passed by assigning this dict.